Representable/representable
# fixed corner radius bug

Fixed the bug where some boxes cut off the corner of text.
To test:
- go through random flows on the site and see if any of the buttons look weird
- the survey page is a place where this used to be a noticeable problem 

<img width="551" alt="Screen Shot 2021-06-11 at 10 41 48 AM" src="https://user-images.githubusercontent.com/78825034/121704725-0312ab80-caa2-11eb-94c9-6a12227eb940.png">
